docs/rst/types/data_kind.rst

```diff
@@ -2,8 +2,8 @@
 
 .. _types_data_kind:
 
-DataKind
```

Comment:
I would change the filename to avoid confusion

Reply:
To be left as is.